inspector/scripts/utils.ts

```diff
@@ -13,3 +12,76 @@ export function getKeyHash() {
     throw new Error(`Key Hash is not defined for network: ${networkName}`);
   }
 }
+
+export async function readAll(inspectorConsumer: Contract) {
+  const { sRandomWord, vrfRequestId } = await readVrf(inspectorConsumer);
+  const { sResponse, rrRequestId } = await readRr(inspectorConsumer);
+  return { sResponse, sRandomWord, rrRequestId, vrfRequestId };
+}
+
+export async function readVrf(inspectorConsumer: Contract) {
+  const sRandomWord = await inspectorConsumer.sRandomWord();
+  const vrfRequestId = await inspectorConsumer.vrfRequestId();
+  return { sRandomWord, vrfRequestId };
+}
```

Comment:
Let's keep an empty line between function definitions. I think there should be some plugin that fixes this for you 🙏